db/fixtures/customization_templates.yml

```diff
@@ -274,3 +274,10 @@
     />\n</unattend>"
   :type: CustomizationTemplateSysprep
   :system: true
+
+- :name: Basic root pass template
+  :description: This template takes use of rootpassword defined in the UI
+  :script: "#cloud-config\nchpasswd:\n  list: |\n    root:<%= MiqPassword.decrypt(evm[:root_password]) %>\n  expire: False"
```

Comment:
my backup suggestion is `MiqPassword.try_decrypt`. the api may have decrypted `evm[:root_password]` already